akka-http-core/src/test/scala/akka/http/scaladsl/TestServer.scala

```diff
@@ -23,10 +23,17 @@ object TestServer extends App {
     akka.loglevel = INFO
     akka.log-dead-letters = off
     akka.stream.materializer.debug.fuzzing-mode = off
+    akka.actor.serialize-creators = off
+    akka.actor.serialize-messages = off
+    akka.actor.default-dispatcher.throughput = 1000
     """)
   implicit val system = ActorSystem("ServerTest", testConf)
-  implicit val fm = ActorMaterializer()
 
+  val settings = ActorMaterializerSettings(system)
+    .withFuzzing(false)
```

Comment:
no need for this becayse `     akka.stream.materializer.debug.fuzzing-mode = off
`